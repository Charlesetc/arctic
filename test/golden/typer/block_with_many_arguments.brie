define main [ : a b c d [ a ]
]
