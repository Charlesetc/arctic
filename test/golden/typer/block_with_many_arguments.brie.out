root[(ident(define):: ident(main):: block[][(block[ident(a)::, ident(b)::, ident(c)::, ident(d)::][(ident(a)::)::]::Function)::Function]::Function)::]::